src/Process/LayeredMaterialNodeProcessing.js

```diff
@@ -283,13 +290,12 @@ export function updateLayeredMaterialNodeImagery(context, layer, node) {
                 if (__DEBUG__) {
                     console.warn('Imagery texture update error for', node, err);
                 }
+
                 const definitiveError = node.layerUpdateState[layer.id].errorCount > MAX_RETRY;
-                node.layerUpdateState[layer.id].failure(Date.now(), definitiveError);
-                if (!definitiveError) {
```

Comment:
Why did you remove this? There's no need to schedule a `notifyChange()` if the `definitiveError` state has been reached.